src/components/viewer/ViewerItem.tsx

```diff
@@ -60,23 +63,18 @@ export default class ViewerItem extends React.Component<Partial<IViewerItemProps
       ViewerItem.editControlDisappearTimeout,
     );
   }
+  onEditClick = (e) => {
```

Comment:
btw. in React there's a "custom" that event handlers are called `handleXY`, e.g. `onClick={handleClick}`, etc.

Reply:
I wasn't sure about that, thanks.